src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
> Jit-EE call getIntConfigValue()/getStringConfigValue() in turn calling CLRConfig::GetConfigValues() with a config option of CLRConfig::REGUTIL_default, which means ignore app config file while reading config value.
>
> The fact now RyuJIT going to read all config values using this API will introduce back compat issue on desktop. For e.g. complus_UseLegacyJIt is a setting we allow even to come from an app config file. Post this change RyuJIt will no longer honor this setting in app config files. There are numerous such clrconfig values.

There are only two policies in use by JIT-specific configuration points: `EEConfig_default` and `REGUTIL_default`. The former is the more permissive of the two, as it allows config values to be fetched from configuration files as well as the registry. I think that something as simple as widening the policy used by the VM's implementation of `getXConfigValue()` from `REGUTIL_default` to `EEConfig_default` is sufficient to address any compatibility issues without adding policy to the API.

/cc @russellhadley @BruceForstall @CarolEidt @AndyAyersMS